TRAMS-API/Repositories/Interfaces/IProjectsRepository.cs

```diff
@@ -19,5 +18,7 @@ public interface IProjectsRepository
         public Task<RepositoryResult<Guid?>> InsertProject(PostAcademyTransfersProjectsD365Model project);
 
         public Task<RepositoryResult<AcademyTransfersProjectAcademy>> GetProjectAcademyById(Guid id);
+
+        public Task<RepositoryResult<Guid?>> UpdateProjectAcademy(Guid id, PatchProjectAcademiesD365Model model);
```

Comment:
Suggestion: Return the updated object.  D365 may run processes following a record update that could in theory change some of the fields on the object.  The consumer wouldn't be aware of these and may continue to use the previous version of the object.  Not essential but just a thought.  May not be important given the limited number of fields on the model.

Reply:
Just noticed this is handled in the API controller so you can ignore this.